tests/common.go

```diff
@@ -186,6 +199,157 @@ func InitInstance() {
 		err = Inst().Backup.Init(Inst().S.String(), Inst().N.String(), Inst().V.String(), token)
 		expect(err).NotTo(haveOccurred())
 	}
+	if Inst().TestRailMilestoneName != "" {
+		client = testrail.NewClient(Inst().TestRailHostname, Inst().TestRailUsername, Inst().TestRailPassword, true)
+		CreateMilestoneIfNotExists()
+	} else {
+		logrus.Infof("No milestone provided, skipping updates to testrail")
+	}
+}
+
+func GetMilestoneByProjectId(projectId int) int {
```

Comment:
Move all Testrail related functions into pkg/testrail folder, then code will be more organized in this case